### PR TITLE
BUG: Make more strings translatable

### PR DESCRIPTION
**Pull request by mhdiop**
* Slicer/SlicerLanguagePacks#21

### CODE REVIEW

File: Base/Python/slicer/ScriptedLoadableModule.py [hunk 1]
```diff
@@ -65,7 +67,7 @@ def getDefaultModuleDocumentationLink(self, docPage=None):
             url = slicer.app.documentationBaseUrl + docPage
         else:
             url = slicer.app.moduleDocumentationUrl(self.moduleName)
-        linkText = f'<p>For more information see the <a href="{url}">online documentation</a>.</p>'
+        linkText = '<p>' + _('For more information see the {tag}online documentation').format(tag=f'<a href="{url}">') + '</a>.</p>'
```
Comment (lassoan) [inline Base/Python/slicer/ScriptedLoadableModule.py:70]:
```suggestion
        linkText = '<p>' + _('For more information see the {url_start}online documentation{url_end}.').format(url_start=f'<a href="{url}">', url_end="</a>") + '</p>'
```

File: Modules/Scripted/DataProbe/DataProbe.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ class DataProbe(ScriptedLoadableModule):
     def __init__(self, parent):
         ScriptedLoadableModule.__init__(self, parent)
 
-        parent.title = "DataProbe"
+        parent.title = _("DataProbe")
```
Comment (lassoan) [inline Modules/Scripted/DataProbe/DataProbe.py:23]:
```suggestion
        parent.title = _("Data Probe")
```

**Comment by mhdiop** [comment]
Thanks @lassoan for your feedback. The updates are done...

**Review by lassoan — APPROVED** [review]
The changes looked good. Made some tweaks, resolved the merge conflicts, it should be ready to go.

**Pull request merged**